Source/Globals.cpp

```diff
@@ -59,7 +61,23 @@ namespace android::global
     void Initialize(JavaVM* javaVM, jobject context)
     {
         g_javaVM = javaVM;
-        g_appContext = GetEnvForCurrentThread()->NewGlobalRef(android::content::Context{context}.getApplicationContext());
+        JNIEnv* env{GetEnvForCurrentThread()};
+        g_appContext = env->NewGlobalRef(android::content::Context{context}.getApplicationContext());
+        // initialize java websocket class here and cache it
+        jclass webSocketClass{env->FindClass("com/jsruntimehost/unittests/WebSocket")};
```

Comment:
We discussed offline, but maybe it would be better to pass in the class from the constructor.